dxr/plugins/extmatch.py

```diff
@@ -0,0 +1,71 @@
+"""Extmatch - Match files in a given folder by extension and provide a link
+in the nav panel of each to the other.
+
+Currently we match "Header" file extensions with "Implementation" file
+extensions.
+"""
+
+from fnmatch import fnmatchcase
+from os.path import relpath, join, basename, splitext, isfile
+from collections import namedtuple
+
+from dxr.indexers import (FileToIndex as FileToIndexBase,
+                          TreeToIndex as TreeToIndexBase)
+from dxr.mime import icon
+from dxr.utils import browse_file_url
+
+# A tuple of extensions along with a title describing the type of extensions:
+_TitledExts = namedtuple('_TitledExts', ['exts', 'title'])
+
```

Comment:
whitespace nit: " Surround top-level function and class definitions with two blank lines. "